staging/src/k8s.io/pod-security-admission/policy/check_apparmor.go

```diff
@@ -56,31 +56,30 @@ func CheckAppArmorProfile() Check {
 }
 
 func allowedProfile(profile string) bool {
-	return profile == corev1.AppArmorBetaProfileRuntimeDefault ||
+	return len(profile) == 0 ||
+		profile == corev1.AppArmorBetaProfileRuntimeDefault ||
 		strings.HasPrefix(profile, corev1.AppArmorBetaProfileNamePrefix)
 }
 
 func appArmorProfile_1_0(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec) CheckResult {
-	forbiddenValues := sets.NewString()
-
-	// undefined is an allowed value for 'container.apparmor.security.beta.kubernetes.io/*'
-	if len(podMetadata.Annotations) == 0 {
-		return CheckResult{Allowed: true}
-	}
-
+	var forbiddenValues []string
 	for k, v := range podMetadata.Annotations {
 		if strings.HasPrefix(k, corev1.AppArmorBetaContainerAnnotationKeyPrefix) && !allowedProfile(v) {
-			forbiddenValues.Insert(fmt.Sprintf("%s:%s", k, v))
+			forbiddenValues = append(forbiddenValues, fmt.Sprintf("%s=%q", k, v))
 		}
 	}
-
 	if len(forbiddenValues) > 0 {
+		sort.Strings(forbiddenValues)
+		var reason string
+		if len(forbiddenValues) == 1 {
+			reason = "forbidden AppArmor profile"
+		} else {
+			reason = "forbidden AppArmor profiles"
+		}
```

Comment:
nit: use pluralize

Reply:
will do... didn't exist when I wrote this one  :)